datafusion/functions/src/string/bit_length.rs

```diff
@@ -39,13 +39,8 @@ impl Default for BitLengthFunc {
 
 impl BitLengthFunc {
     pub fn new() -> Self {
-        use DataType::*;
         Self {
-            signature: Signature::uniform(
-                1,
-                vec![Utf8, LargeUtf8],
-                Volatility::Immutable,
-            ),
+            signature: Signature::string(1, Volatility::Immutable),
```

Comment:
The function declared it accepts `Utf8, LargeUtf8`. Now it accepts "any string".

Does this function work for string views?
Is this covered with tests?